packages/compass-indexes/src/components/type-column/type-column.jsx

```diff
@@ -22,70 +39,52 @@ class TypeColumn extends PureComponent {
     openLink: PropTypes.func.isRequired,
   };
 
-  _textTooltip() {
+  canRenderTooltip() {
+    return (
+      ['text', 'wildcard', 'columnstore'].indexOf(this.props.index.type) !== -1
+    );
+  }
+
+  renderTooltip() {
     const info = pick(this.props.index.extra, [
       'weights',
       'default_language',
       'language_override',
       'wildcardProjection',
       'columnstoreProjection',
     ]);
-    return map(info, (v, k) => {
-      return format('%s: %j', k, v);
-    }).join('<br />');
-  }
-
-  /**
-   * Render the type div.
-   *
-   * @returns {React.Component} The type div.
-   */
-  renderType() {
-    let tooltipOptions = {};
-    if (
-      this.props.index.type === 'text' ||
-      this.props.index.type === 'wildcard' ||
-      this.props.index.type === 'columnstore'
-    ) {
-      const tooltipText = `${this._textTooltip()}`;
-      tooltipOptions = {
-        'data-tip': tooltipText,
-        'data-for': TOOLTIP_ID,
-        'data-effect': 'solid',
-        'data-multiline': true,
-        'data-border': true,
-      };
-    }
-
-    const helpLink = getIndexHelpLink(this.props.index.type.toUpperCase());
-
-    return (
-      <div
-        {...tooltipOptions}
-        className={styles[`type-column-property-${this.props.index.type}`]}
-        data-test-id="index-field-type"
-      >
-        {this.props.index.type}
-        {helpLink && (
-          <InfoSprinkle
-            helpLink={helpLink}
-            onClickHandler={this.props.openLink}
-          />
-        )}
-      </div>
-    );
+    const items = map(info, (v, k) => {
+      return <Body>{`${k}: ${JSON.stringify(v)}`}</Body>;
+    });
+    return <>{items}</>;
   }
 
-  /**
-   * Render the type column.
-   *
-   * @returns {React.Component} The type column.
-   */
   render() {
+    const helpLink = getIndexHelpLink(this.props.index.type.toUpperCase());
     return (
-      <td className={styles['type-column']}>
-        {this.renderType()}
-        <ReactTooltip id={TOOLTIP_ID} data-testid="tooltip-index-field-type" />
+      <td data-testid="index-field-type">
+        <Tooltip
+          data-testid="tooltip-index-field-type"
+          enabled={this.canRenderTooltip()}
+          trigger={({ children, ...props }) => (
+            <span {...props}>
+              {children}
+              <Badge variant={BadgeVariant.DarkGray} className={badgeStyles}>
```

Comment:
Previously we were showing different colors for the badges, are we planning to re-add them?
before:
![Screen Shot 2022-08-15 at 4 50 09 PM](https://user-images.githubusercontent.com/1791149/184716115-6f51e2b3-f441-419d-a524-1a860d5f3b56.png)

after:
![Screen Shot 2022-08-15 at 4 50 04 PM](https://user-images.githubusercontent.com/1791149/184716093-f1e549b0-fec0-4b67-80bd-10252e587bc1.png)

Reply:
its be design that we are removing coloured badges.

Reply:
Ah gotcha, I'll miss them 😿 
Looks good though.